managed-services/webhosting/how-to/order-dedicated-ip.mdx

```diff
@@ -0,0 +1,67 @@
+* **SSL/TLS certificates:** If you want to secure your website with an SSL/TLS certificate, having a dedicated IP address can be necessary. While it is possible to use Server Name Indication (SNI) to enable SSL on shared IP addresses, not all older browsers support SNI. In such cases, a dedicated IP address becomes necessary for compatibility.
+* **Avoiding blacklisting:** Shared IP addresses can be more susceptible to being blacklisted if another user on the same server engages in spam or other malicious activities. With a dedicated IP, you have more control over the reputation of your IP address.
+
+<Message type="note">
+  Needing a dedicated IP address can depend on your specific requirements and the nature of your website. Many websites function perfectly well with shared IP addresses, and dedicated IPs are often more critical for specific scenarios, such as e-commerce websites that require SSL/TLS or those with unique server configurations.
+  Always assess your specific needs before deciding whether a dedicated IP is necessary for your shared hosting environment.
+</Message>
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [registered a domain name](/network/domains-and-dns/how-to/register-internal-domain/) at Scaleway or another registrar
+  - You have a Web Hosting plan with a shared IP address
+</Message>
+
+## Activate a dedicated IP address for a Web Hosting plan
+
+1. Navigate to the **Managed Services** section of the [console](https://console.scaleway.com/) side menu and click **Web Hosting**. This action will lead you to the **Web Hosting** overview page.
+2. Click either <Icon name="seeMore" /> or click on the domain name associated with the Web Hosting you want to configure. This will take you to the **Hosting Information** page.
+3. Within the hosting options area of your Web Hosting, click on **Manage Options**.
+4. Activate the **Dedicated IP** toggle to initiate the process of acquiring a dedicated IP address for your Web Hosting plan.
+5. Complete the process by clicking **Update Web Hosting Plan**. 
+    This action will not only update the configuration of your Web Hosting but also assign the newly acquired dedicated IP to it.
+
+<Message type="tip">
+  Alternatively, you can activate this option directly while configuring a new Web Hosting plan.
+</Message>
+
+## Deactivate a dedicated IP address for a Web Hosting plan
+
+1. Navigate to the **Managed Services** section of the [console](https://console.scaleway.com/) side menu and click **Web Hosting**. This action will lead you to the **Web Hosting** overview page.
+2. Click either <Icon name="seeMore" /> or click on the domain name associated with the Web Hosting you want to configure. This will take you to the **Hosting Information** page.
```

Comment:
```suggestion
2. Click either <Icon name="seeMore" /> or the domain name associated with the Web Hosting you want to configure. This will take you to the **Hosting Information** page.
```